File: Javascript/Kalkulator Sederhana.js
function kalkulator() {
    console.log("Selamat datang di Kalkulator Sederhana!");
    console.log("Operasi yang didukung: +, -, *, /");

    // Baris 6 dan 8:
    const num1 = Number.parseFloat(prompt("Masukkan angka pertama:"));
    const num2 = Number.parseFloat(prompt("Masukkan angka kedua:"));

    // Baris 10:
    if (Number.isNaN(num1) || Number.isNaN(num2)) {
        console.log("Masukkan angka yang valid!");
        // ...
    }

    while (true) {
        const operator = prompt("Masukkan operator (+, -, *, /):");
        if (!['+', '-', '*', '/'].includes(operator)) {
            console.log("Operator tidak valid! Silakan coba lagi.");
            continue;
        }

        let hasil;
        switch (operator) {
            case '+':
                hasil = num1 + num2;
                break;
            case '-':
                hasil = num1 - num2;
                break;
            case '*':
                hasil = num1 * num2;
                break;
            case '/':
                if (num2 === 0) {
                    console.log("Tidak bisa membagi dengan nol!");
                    continue;
                }
                hasil = num1 / num2;
                break;
            default:
                console.log("Operator tidak valid!");
                continue;
        }

        console.log(`Hasil: ${num1} ${operator} ${num2} = ${hasil}`);

        const lagi = prompt("Apakah ingin menghitung lagi? (y/n):");
        if (lagi.toLowerCase() !== 'y') break;
    }
}

// Jalankan kalkulator
kalkulator();

File: Python/Kalkulator Sederhana.py
# Kalkulator Sederhana
# Program ini menjalankan operasi matematika dasar: +, -, *, /

def kalkulator():
    print("Selamat datang di Kalkulator Sederhana!")
    print("Operasi yang tersedia: +, -, *, /")
    
    while True:
        try:
            # Input angka pertama
            num1 = float(input("Masukkan angka pertama: "))
            
            # Input operator
            operator = input("Masukkan operator (+, -, *, /): ")
            
            # Input angka kedua
            num2 = float(input("Masukkan angka kedua: "))
            
            # Hitung berdasarkan operator
            if operator == '+':
                hasil = num1 + num2
            elif operator == '-':
                hasil = num1 - num2
            elif operator == '*':
                hasil = num1 * num2
            elif operator == '/':
                if num2 == 0:
                    print("Error: Pembagian dengan nol tidak diperbolehkan!")
                    continue
                hasil = num1 / num2
            else:
                print("Operator tidak valid! Gunakan +, -, *, atau /.")
                continue
            
            # Tampilkan hasil
            print(f"Hasil: {num1} {operator} {num2} = {hasil}")
            
            # Tanya apakah ingin lanjut
            lagi = input("Apakah ingin menghitung lagi? (y/n): ").lower()
            if lagi != 'y':
                print("Terima kasih telah menggunakan kalkulator!")
                break
        
        except ValueError:
            print("Input tidak valid! Masukkan angka yang benar.")

# Jalankan program
if __name__ == "__main__":
    kalkulator()